pkg/env/env.go

```diff
@@ -0,0 +1,5 @@
+package env
```

Comment:
Not quite sure about the name of the package. I guess it'll be more clear once we have more variables to configure.